### PR TITLE
Dts 29349 : DIR, FTPR KPI label level field mapping

### PR DESCRIPTION
**Pull request by eswbogol**
For DIR and FTPR KPI's, a new label mapping is included, when a label is provided the kpi's will populate the data based on the issue types included and the labels mentioned.

### CODE REVIEW

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/mongock/rollback/release_810/DIRFTPRLabelsChangeLog.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+import com.mongodb.client.model.Filters;
+
+import io.mongock.api.annotations.ChangeUnit;
+import io.mongock.api.annotations.Execution;
+import io.mongock.api.annotations.RollbackExecution;
+
+/**
+ * @author eswbogol
+ */
+@ChangeUnit(id = "r_DIR_FTPR_Labels_ChangeLog", order = "08103", author = "eswbogol", systemVersion = "8.1.0")
```
Comment (shunaray) [inline customapi/src/main/java/com/publicissapient/kpidashboard/apis/mongock/rollback/release_810/DIRFTPRLabelsChangeLog.java:31]: pls mention this in confluence also, mongo id should be in small letters without camel-casing and seperated by underscore

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/mongock/upgrade/release_810/DIRFTPRLabelsChangeLog.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+import org.springframework.data.mongodb.core.MongoTemplate;
+
+import io.mongock.api.annotations.ChangeUnit;
+import io.mongock.api.annotations.Execution;
+import io.mongock.api.annotations.RollbackExecution;
+
+/**
+ * @author eswbogol
+ */
+@ChangeUnit(id = "DIR_FTPR_Labels_ChangeLog", order = "8103", author = "eswbogol", systemVersion = "8.1.0")
```
Comment (shunaray) [inline customapi/src/main/java/com/publicissapient/kpidashboard/apis/mongock/upgrade/release_810/DIRFTPRLabelsChangeLog.java:30]: mongo id should be in small letters without camel-casing and seperated by underscore

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/mongock/rollback/release_810/DIRFTPRLabelsChangeLog.java (hunk 1 above)
Comment (eswbogol) [reply customapi/src/main/java/com/publicissapient/kpidashboard/apis/mongock/rollback/release_810/DIRFTPRLabelsChangeLog.java:31]: done

File: customapi/src/main/java/com/publicissapient/kpidashboard/apis/mongock/upgrade/release_810/DIRFTPRLabelsChangeLog.java (hunk 2 above)
Comment (eswbogol) [reply customapi/src/main/java/com/publicissapient/kpidashboard/apis/mongock/upgrade/release_810/DIRFTPRLabelsChangeLog.java:30]: done

**Review by shunaray — DISMISSED** [review]

**Review by gipathak — APPROVED** [review]

**Review by gipathak — APPROVED** [review]

**Review by Chittauri — APPROVED** [review]

**Review by gipathak — APPROVED** [review]

**Pull request merged**